### PR TITLE
Ad implementation

### PR DESCRIPTION
**Pull request by asimqt**

### CODE REVIEW

File: app/isomorphic/components/story-templates/blank.js [hunk 1]
```diff
@@ -29,20 +29,35 @@ function BlankStoryTemplate(props) {
       <div className="blank-story--wrapper">
         <div className="blank-story--content">
           <StoryHeaderCard story={props.story}/>
-          {props.story.cards.map((card, index) => <StoryPageCard key={index} card={card} story={props.story}/>)}
+          {props.story.cards.map((card, index) => {
+            if(index === 0) {
+              return <div className="blank-story__card--with-ad">
+                <StoryPageCard key={index} card={card} story={props.story}/>
```
Comment (ajaymenon0) [inline app/isomorphic/components/story-templates/blank.js]: If possible don't use index as the key.
Check this:
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

File: app/assets/stylesheets/layout/_text-story.scss [hunk 2]
```diff
@@ -11,6 +11,15 @@
   }
   &--content {
     margin: -70px auto 20px;
+    background-color: $white;
+    padding: 15px;
+    z-index: 10;
+    @include breakpoint($pc){
+      padding: 30px 220px 55px 55px;
+      min-height: 1300px;
```
Comment (rvgpl) [inline app/assets/stylesheets/layout/_text-story.scss]: Why do you need this?

File: app/isomorphic/components/pages/home.js [hunk 3]
```diff
@@ -37,9 +37,13 @@ class HomePage extends React.Component {
             'collection-color': collection.color,
             'collection-section': _.get(collection,['metadata', 'section'], [])
           }
+          const componentProps = {stories: collection.items, key: collection.id, config: config}
+          if(collection['associated-metadata']['layout'] === 'FullscreenMediaList') {
+            componentProps.adSlot= true
```
Comment (rvgpl) [inline app/isomorphic/components/pages/home.js]: Are you adding a new key called `adSlot` to a const variable? Does this work?

File: app/assets/stylesheets/layout/_text-story.scss (hunk 2 above)
Comment (asimqt) [reply app/assets/stylesheets/layout/_text-story.scss]: When the story doesn't have a minimum content, related stories were overlapping with the ad and metype widget. I couldn't think of any other way which will give me a decent design

File: app/isomorphic/components/pages/home.js (hunk 3 above)
Comment (asimqt) [reply app/isomorphic/components/pages/home.js]: Yes. It was working. 😺 Let me check

Comment (asimqt) [reply app/isomorphic/components/pages/home.js]: It's okay to change the props. I think as we are not changing the actual reference this is okay.

**Pull request merged**